### ISSUE TITLE
Listing possible new providers

### ISSUE THREAD

**Issue by frizensami** [frizensami/crowdwise#87]
- Bing Search API: https://www.microsoft.com/en-us/bing/apis/pricing
- Arxiv API

**Comment by yshalsager** [comment]
https://lobste.rs/ is another great place to get comments from, it's link aggregating website like hackernews.

**Comment by frizensami** [comment]
Good idea! Seems like it doesn't have a direct API, so we have to do some scraping.

**Comment by yshalsager** [comment]
@frizensami There is no API at but you can get JSON representations of various pages by appending dot JSON to page URL.
